spec/controllers/dams_authorities_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe DamsAuthoritiesController do
```

Comment:
Are these controller tests giving us additional test coverage that the feature tests are not? This is admittedly influenced by the recent trend of [Rails 5 removing controller test support out of the box](https://medium.com/table-xi/whats-up-with-rails-controller-tests-f0ece1fdd9f0#.of8ckh2qd). But honestly, I'm just curious because if there's anything we can do to make our test suite faster, while still maintaining coverage, I'm all for it